src/Robo/Commands/Setup/BuildCommand.php

```diff
@@ -134,4 +134,45 @@ public function installAlias() {
       ->run();
   }
 
+  /**
+   * Checks the repo if there is modified files.
+   */
+  public function checkCleanRepo() {
+    $clean = $this->taskExec("git status --porcelain")
+      ->dir($this->getConfigValue('repo.root'))
+      ->interactive()
+      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
+      ->run();
+    return $clean;
+  }
+
+  /**
+   * Adds a project dependency and ensures configuration management is in sync.
+   *
+   * @command add-dependency
```

Comment:
Please document the argument like this:
`* @param integer $one The first parameter.`